token-lending/program/src/state/mod.rs

```diff
@@ -31,7 +31,7 @@ pub const UNINITIALIZED_VERSION: u8 = 0;
 
 /// Number of slots per year
 pub const SLOTS_PER_YEAR: u64 =
-    DEFAULT_TICKS_PER_SECOND / DEFAULT_TICKS_PER_SLOT * SECONDS_PER_DAY * 365;
+    DEFAULT_TICKS_PER_SECOND * SECONDS_PER_DAY * 365 / DEFAULT_TICKS_PER_SLOT;
```

Comment:
Ugh, need `const_panic` stabilized so we can just switch all of these to checked math.

Reply:
Oooh yeah that'd be great!  I also didn't realize all of checked math is const

Reply:
`const fn` all the things!